### PR TITLE
Install bazelisk for target architecture

### PR DESCRIPTION
**Pull request by prem2012**
Installing the correct bazel binary for the target architecture

### CODE REVIEW

File: docker/build.sh [hunk 1]
```diff
@@ -2,7 +2,20 @@
 
 set -e
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+ARCHITECTURE="$(uname -m)"
 DOCKER_IMAGE_NAME="trellis-docker"
 
+case $ARCHITECTURE in
+    x86_64)
+        DOCKER_IMAGE_NAME+="amd64";;
```
Comment (bsirang) [inline docker/build.sh:11]: Looks like the colon is missing here. Needs to be `:amd64`. The intent is to make a string like `trellis-docker:amd64` which is `name:tag`

File: docker/shell.sh [hunk 2]
```diff
@@ -5,11 +5,25 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 WORKSPACE_ROOT="$SCRIPT_DIR/.."
 HOME_DIR="/root/trellis"
 
+ARCHITECTURE="$(uname -m)"
+DOCKER_IMAGE_NAME="trellis-docker"
+
+case $ARCHITECTURE in
+    x86_64)
+        DOCKER_IMAGE_NAME+="amd64";;
```
Comment (bsirang) [inline docker/shell.sh:13]: Similar comment as above

**Review by bsirang — COMMENTED** [review]
Thanks! Had a small comment.

**Review by bsirang — APPROVED** [review]

File: docker/build.sh (hunk 1 above)
Comment (prem2012) [reply docker/build.sh]: resolved!

**Pull request merged**